packages/eas-cli/src/build/createContext.ts

```diff
@@ -48,6 +49,7 @@ export async function createBuildContextAsync<T extends Platform>({
   message?: string;
   actor: Actor;
   graphqlClient: ExpoGraphqlClient;
+  analyticsManager: IAnalyticsManager;
```

Comment:
Not a strong opinion but maybe we could rename this from `analyticsManager` to `analytics`? To me, the word "manager" in code is almost always meaningless. The interface name can stay as it is.